docs/content/observability/tracing/opentelemetry.md

```diff
@@ -242,5 +242,5 @@ tracing:
 ```
 
 ```bash tab="CLI"
---tracing.openTelemetry.grpc=true
+--tracing.openTelemetry.grpc
```

Comment:
```suggestion
--tracing.openTelemetry.grpc=true
```

Reply:
It would be nicer to be as consistent as possible.

@ldez Can you detail why it should be a boolean in CLI and an empty struct in YAML ?

Reply:
because an empty struct is a boolean inside the CLI. The CLI is not JSON or YAML.

`--tracing.openTelemetry.grpc` is equal to `--tracing.openTelemetry.grpc=true` => a boolean value.

Reply:
I understand from your explanation that both are working. That's also what I observed in my tests.

A real fix for this confusion could be to align CLI struct with JSON and YAML. FTM, I don't know how to make this kind of fix.

As a workaround, since they are equal and one of them, `--tracing.openTelemetry.grpc=true`, can make uninformed user believe a boolean is expected, the other is not, then why not helping users to be less confused ?

Reply:
CLI cannot be aligned with JSON and YAML because CLI uses simple values and not objects.

An empty struct is not `struct{}` (it's just a specific case), it's a struct that has fields but the fields are not filled.
For example, you need to enable Docker but using the default configuration.

Can you accept my suggestion?

Reply:
I would be happy to accept your suggestion, you often have interesting point of view.

I understand that aligning CLI conf with TOML and YAML is subtle and delicate.

Since `--tracing.openTelemetry.grpc` is less confusing and works, I do not understand why you are suggesting to keep the more confusing `--tracing.openTelemetry.grpc=true` in the doc ?

Reply:
We always document the boolean option (real boolean or empty struct) with an explicit value to be homogenous and avoid confusion for real CLI users.

Reply:
:thinking: What is a real CLI user ? Is there false CLI user ? I do not understand what that means.

I agree with you that there is value to be homogenous. 

Since the boolean option in CLI can represent a real boolean or an empty struct, wdyt about:

* documenting all _real boolean_ conf on CLI with `--xx.yy=true`
* documenting all _empty struct_ conf on CLI with `--xx.yy`